news/9039.doc.rst

```diff
@@ -0,0 +1 @@
+adding a section to pip user guide to cover pip backtracking, connected to #8975.
```

Comment:
Most of these details are implied by being in pip's Changelog page. Reference to #8975 isn't required since users can reach that from the linked issue.

Suggested rewording:

```suggestion
Add a section to the User Guide to cover backtracking during dependency resolution.
```